src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -472,6 +471,7 @@ public WorkspaceViewModel(WorkspaceModel model, DynamoViewModel dynamoViewModel)
             Model.NoteRemoved += Model_NoteRemoved;
             Model.NotesCleared += Model_NotesCleared;
 
+            Model.AnnotationsAdded += Model_AnnotationsAdded;
```

Comment:
maybe a comment on the difference between these events and where they might be triggered?

Reply:
removed since it does not help performance that much (at least in this case - when loading a workspace)